deck.gl/docs/guides/01-getting-started.md

```diff
@@ -1,205 +1,49 @@
 ## Getting started
 
-In this guide, you will learn the basics of visualizing data with the Web SDK on top of a world [basemap](https://carto.com/help/glossary/#basemap). There are no previous requirements to complete this guide, but a basic knowledge of HTML, CSS and JavaScript would be helpful.
+In this guide, you will learn how to install CARTO for deck.gl in your computer and run the examples.
 
-After completing this guide, you will have your first Web SDK map!
+### Installation
 
-<div class="example-map">
-    <iframe
-        id="getting-started-final-result"
-        src="/developers/web-sdk/examples/maps/guides/getting-started/step-3.html"
-        width="100%"
-        height="500"
-        frameBorder="0">
-    </iframe>
-</div>
+You can work with CARTO for deck.gl with the NPM module for React or Pure JS applications:
 
-### Basic setup
+`npm install @deck.gl`
 
-The most straight-forward way to use the Web SDK is to include the required files from our CDN as seen in the code below. The Web SDK is based on the powerful deck.gl library. In addition to the Web SDK JavaScript file, you need to add:
+`npm install @deck.gl/carto`
 
-* deck.gl JavaScript file
-* Mapbox GL JavaScript and CSS files
+Or you can use the scripting API:
 
-```html
-    <!-- Include deck.gl from unpkg CDN -->
-    <script src="https://unpkg.com/deck.gl@8.2.0/dist.min.js"></script>
+`<script src="https://unpkg.com/deck.gl@latest/dist.min.js"></script>`
 
-    <!-- Include Mapbox GL from the Mabpox CDN -->
-    <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js'></script>
-    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css' rel='stylesheet' />
+`<script src="https://unpkg.com/deck.gl@latest/carto/dist.min.js"></script>`
```

Comment:
This url is currently failing, as it is trying to resolve this: https://unpkg.com/deck.gl@8.2.8/carto/dist.min.js

I think we should use: **https://unpkg.com/@deck.gl/carto@latest/dist.min.js** (indeed I think it's a good idea to replace the 8.3.0-alpha.1 from scripting examples, to point to that one). I can do the later in another PR if you want.

Reply:
Changed in all the scripting examples